pulsar-client/src/main/java/org/apache/pulsar/client/impl/HttpLookupService.java

```diff
@@ -187,7 +186,7 @@ public CompletableFuture<Optional<SchemaInfo>> getSchema(TopicName topicName, by
                             .props(response.getProperties())
                             .build();
                     future.complete(Optional.of(SchemaInfoUtil.newSchemaInfo(schemaName, data)));
-                } catch (IOException err) {
+                } catch (Exception err) {
```

Comment:
There is no exception will be thrown. The try-catch can be removed.

Reply:
addressed